api/stats/internal.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package nighthawk;
+
+// Internal protos for representing Statistics in a way that allows
+// "lossless" reconstruction. Do not consume / rely on this in any other place
+// than Nighthawk code as this may be subject to change.
+
+message InternalSimpleStatistic {
+  uint64 count = 1;
```

Comment:
Were some of these meant to be oneoff? Or is this message going to carry multiple values?

Reply:
I since saw how they are used, so resolving this.